iotilegateway/iotilegateway/supervisor/service_tile.py

```diff
@@ -65,7 +69,13 @@ def call_rpc(self, rpc_id, payload=bytes()):
         # run time of RPCs that could be long running.  The caller of the RPC
         # through the tile will know what an appropriate timeout is for the
         # RPC that they are trying to call.
-        resp = self._client.send_rpc(self._service, rpc_id, payload, timeout=120.0)
+        if inspect.iscoroutinefunction(self._client.send_rpc):
```

Comment:
I worry about leaving this in here since we ended up not deciding to use the asyncio event loop for this tile at this point.  I would rather leave this tile as it is because when we come back and port over to asyncio for real it will likely be with a different base class like `AsyncVirtualTile` where `call_rpc` itself is a coroutine so this isn't needed.